packages/teleport/src/components/MfaDeviceList/MfaDeviceList.tsx

```diff
@@ -34,80 +26,56 @@ export default function MfaDeviceList({
   remove,
   mostRecentDevice,
   mfaDisabled = false,
-  ...styles
+  isSearchable = false,
+  style,
 }: Props) {
-  const [sortDir, setSortDir] = useState<Record<string, string>>({
-    registeredDate: SortTypes.ASC,
-  });
-
-  function sort(data) {
-    const columnKey = Object.getOwnPropertyNames(sortDir)[0];
-    const sorted = sortBy(data, columnKey);
-    if (sortDir[columnKey] === SortTypes.ASC) {
-      return sorted.reverse();
-    }
-
-    return sorted;
-  }
-
-  function onSortChange(columnKey: string, sortDir: string) {
-    setSortDir({ [columnKey]: sortDir });
-  }
-
-  const data = sort(devices);
-
   return (
-    <StyledTable data={data} {...styles}>
-      <Column
-        columnKey="description"
-        cell={<TextCell />}
-        header={<Cell>Type</Cell>}
-      />
-      <Column
-        columnKey="name"
-        cell={<NameCell />}
-        header={<Cell>Device Name</Cell>}
-      />
-      <Column
-        columnKey="registeredDate"
-        header={
-          <SortHeaderCell
-            sortDir={sortDir.registeredDate}
-            onSortChange={onSortChange}
-            title="Registered"
-          />
-        }
-        cell={<DateCell />}
-      />
-      <Column
-        columnKey="lastUsedDate"
-        header={
-          <SortHeaderCell
-            sortDir={sortDir.lastUsedDate}
-            onSortChange={onSortChange}
-            title="Last Used"
-          />
-        }
-        cell={<DateCell />}
-      />
-      <Column
-        header={<Cell />}
-        cell={
-          <RemoveCell
-            remove={remove}
-            mostRecentDevice={mostRecentDevice}
-            mfaDisabled={mfaDisabled}
-          />
-        }
-      />
-    </StyledTable>
+    <StyledTable
```

Comment:
Does it still need this `<StyledTable>` ?

Reply:
Yes because here we add some styles that aren't default `vertical-align: middle` and `height: 32px`